apps/scalable/7926_PDS.0.svg

```diff
@@ -0,0 +1,44 @@
+<svg height="48" width="48" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Can you move this file content to `proteus.svg` and then make this file (`7926_PDS.0.svg`) a symlink to `proteus.svg`?